### ISSUE TITLE
App won't start or uninstall

### ISSUE THREAD

**Issue by zstauber** [microsoft/vscode#82036]
- VSCode Version:
- OS Version:

Steps to Reproduce:

1.
2.

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by zstauber** [comment]
Version: no clue, can't open it.  Been keeping it very up-to-date though.
OS Version: Windows 10 with 1903 feature update

Steps to Reproduce:
1. Start app, or try, anyway
2. Try to uninstall so I can reinstall

Either one crashes.  If I try to open, I get
![vscode](https://user-images.githubusercontent.com/6342703/66278356-742da480-e865-11e9-947a-4ecff6bd6a3f.png)

If I try to uninstall it from Add/Remove Programs, I get
![vscode2](https://user-images.githubusercontent.com/6342703/66278380-b525b900-e865-11e9-84aa-7bfb97169d31.png)
This is literally the very first time I've gotten either of these errors.  Using VS Code and uninstalling apps has worked flawlessly until now.  And yes, I did run chkdsk on all my drives and it found no corrupt files.  I have not had a crash lately, certainly not since the last time I ran VS Code successfully, since I use it all the time.  But, maybe it will never let me use it again!
